image-builder/Dockerfile

```diff
@@ -0,0 +1,3 @@
+From fedora:latest
```

Comment:
@jodh-intel I changed to `latest`, I like Dockerfile templates but will take more time to re-work.

Reply:
OK, I guess that's fine for now. Once this lands, could you raise an issue so we don't forget about auto-generating the dockerfile though?

Reply:
sure